src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,36 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate startDate = LocalDate.parse(dateFrom, formatter).minusDays(1);
+        LocalDate finishDate = LocalDate.parse(dateTo, formatter).plusDays(1);
+
+        StringBuilder result = new StringBuilder("Report for period ")
+                .append(dateFrom)
+                .append(" - ")
+                .append(dateTo);
+        int daySalary = 0;
+
+        for (String name : names) {
+            for (String employeeWorkingDayDatum : data) {
+                String[] employeeInfo = employeeWorkingDayDatum.split(" ");
+                LocalDate workDate = LocalDate.parse(employeeInfo[0], formatter);
```

Comment:
add constants for indexes
![image](https://github.com/mate-academy/jv-salary-info/assets/20326858/9d994548-c97e-4bf3-8c50-1e98b97b82e2)